demos/src/subnav.mustache

```diff
@@ -0,0 +1,85 @@
+<header class="o-header" data-o-component="o-header" data-o-header--no-js>
+	{{#nav}}
+	<nav id="o-header-nav-mobile" class="o-header__row o-header__nav o-header__nav--mobile" role="navigation">
+		<ul class="o-header__nav-list">
+			{{#mobile}}
+			<li class="o-header__nav-item">
+				<a class="o-header__nav-link o-header__nav-link--primary" href="{{href}}" aria-selected="{{selected}}">{{name}}</a>
+			</li>
+			{{/mobile}}
+		</ul>
+	</nav>
+
+	<nav id="o-header-nav-desktop" class="o-header__row o-header__nav o-header__nav--desktop" role="navigation">
+		<div class="o-header__container">
+			<ul class="o-header__nav-list o-header__nav-list--left">
+				{{#desktop}}
+				<li class="o-header__nav-item">
+					<a class="o-header__nav-link o-header__nav-link--primary" href="{{href}}" aria-selected="{{selected}}" id="o-header-link-{{index}}">{{name}}</a>
+				</li>
+				{{/desktop}}
+			</ul>
+			<ul class="o-header__nav-list o-header__nav-list--right">
+				{{#isSignedIn}}
+					<li class="o-header__nav-item">
+						<a class="o-header__nav-link o-header__nav-link--utility" href="https://markets.ft.com/data/portfolio/dashboard">Portfolio</a>
+					</li>
+					<li class="o-header__nav-item">
+						<a class="o-header__nav-link o-header__nav-link--utility" href="https://registration.ft.com/registration/selfcare/">My Account</a>
+					</li>
+				{{/isSignedIn}}
+				{{^isSignedIn}}
+					<li class="o-header__nav-item">
+						<a class="o-header__nav-link o-header__nav-link--utility" href="/login">Sign In</a>
+					</li>
+					<li class="o-header__nav-item">
+						<a class="o-header__nav-button" href="/products">Subscribe</a>
+					</li>
+				{{/isSignedIn}}
+			</ul>
+		</div>
+	</nav>
+	{{/nav}}
+
+	{{#subnav}}
+	<div class="o-header__subnav" role="navigation" data-o-header-subnav>
```

Comment:
quite a few elements given `role="navigation"` - I think an `aria-label="..."` added to each should differentiate them